bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -209,7 +214,21 @@ subjects:
   name: ovn-kubernetes-node
   namespace: openshift-ovn-kubernetes
 {{ end }}
-
+{{- if .IsNetworkTypeMigration }}
```

Comment:
Why does the `ovn-kubernetes-node` service account require the whole `openshift-ovn-kubernetes-node-limited` ClusterRole during the migration? I think we should limit it to what is required.
Additionally this binding already exists here: https://github.com/openshift/cluster-network-operator/blob/6569070efa9b09925568d0f321e59f19cb0ff563/bindata/network/ovn-kubernetes/common/002-rbac-node.yaml#L194-L216 (You would need to use the per-node certificate if the webhook is enabled)